modules/piximediaBidAdapter.js

```diff
@@ -0,0 +1,46 @@
+import * as utils from 'src/utils';
+import { registerBidder } from 'src/adapters/bidderFactory';
+
+const BIDDER_CODE = 'piximedia';
+const ENDPOINT = '//ad.piximedia.com/prebid';
+
+export const spec = {
+  code: BIDDER_CODE,
+  isBidRequestValid: function(bid) {
+    return !!(bid.params && bid.params.siteId && bid.params.placementId);
+  },
+  buildRequests: function(validBidRequests) {
+    return validBidRequests.map(bidRequest => {
+      let parseSized = utils.parseSizesInput(bidRequest.sizes);
+      let arrSize = parseSized[0].split('x');
+      return {
+        method: 'GET',
+        url: ENDPOINT,
+        data: {
+          timestamp: utils.timestamp(),
+          pver: '1.0',
+          pbparams: JSON.stringify(bidRequest.params),
+          pbwidth: arrSize[0],
+          pbheight: arrSize[1],
+          pbbidid: bidRequest.bidId,
+        },
+      };
+    });
+  },
+  interpretResponse: function(serverResponse, request) {
+    const res = serverResponse.body;
+    const bidResponse = {
+      requestId: res.bidId,
+      cpm: parseFloat(res.cpm),
+      width: res.width,
+      height: res.height,
+      creativeId: res.creative_id,
```

Comment:
The response to your test parameters is sending back creativeId not creative_id
![screen shot 2018-08-13 at 5 01 16 pm](https://user-images.githubusercontent.com/13356910/44064581-aeb30bda-9f1a-11e8-912e-078a8f7666da.png)

Reply:
Hi @mike-chowla, thanks for noticing!

I've just returned from extended vacation, and I'm sorry I took so long to reply to your comment. 

I have already made sure this issue is fixed on our production system, would you like me to create a new PR or is there a way to reopen this issue?

Thanks ! 👍